docs/includes/google-credentials.md

```diff
@@ -1,24 +1,28 @@
-**Tip:** If you see Google Play Console or Google Developer Console in your local language, add `&hl=en` at the end of the URL (before any `#...`) to switch to English.
+**Tip:** If you see Google Play Console or Google Developer Console in your local language, add `&hl=en` at the end of the URL (before any `#...`) to switch to English. All the links below already have this to make it easier to find the correct buttons.
 
-1. Open the [Google Play Console](https://play.google.com/console/)
+1. Open the [Google Play Console](https://play.google.com/console/?hl=en)
 1. Click **Account Details**, and note the **Developer Account ID** listed there
-1. Click **Setup** → **API access**
-1. Click the **Create new service account** button
-1. Follow the **Google Cloud Platform** link in the dialog, which opens a new tab/window:
+1. Open [Create Project on Google Cloud](https://console.cloud.google.com/projectcreate/?hl=en) and create a project if none exists that fits
+1. Enable the [Google Play Developer API](https://console.developers.google.com/apis/api/androidpublisher.googleapis.com/?hl=en) by selecting the project and pushing "enable"
+1. Open [Service Accounts on Google Cloud](https://console.cloud.google.com/iam-admin/serviceaccounts?hl=en) and select the project you used before
    1. Click the **CREATE SERVICE ACCOUNT** button at the top of the **Google Cloud Platform Console**
```

Comment:
Should this 2nd line (and subsequent lines) still be indented "inside" the "open service accounts on google cloud" bullet? It used to be nested because of a "new tab/window" but this doesn't seem to be the case anymore, it simply opens the creation flow in the same tab/window

Reply:
I reindented it based on which page you are on, what do you think about this?

Reply:
Perfect!